src/server/land-grants/controllers/auth/land-grants-question-with-auth-check.controller.js

```
@@ -12,7 +13,15 @@ export default class LandGrantsQuestionWithAuthCheckController extends QuestionP
   }
 
   performAuthCheck = async (request, h) => {
-    const landParcels = (await fetchParcels(request)) || []
+    let landParcels = []
+    try {
+      landParcels = (await fetchParcels(request)) || []
+    } catch (error) {
+      log(LogCodes.SYSTEM.EXTERNAL_API_ERROR, {
+        endpoint: `Land grants API`,
+        error: `fetch parcel data for auth check: ${error.message}`
+      })
```

If you don't rethrow the error here will users not get  an unauthorised message?  that would hide an error here

---

In case of exception the list of allowed land parcels will be empty so the `this.landParcelBelongsToSbi` check will fail, causing  the rendering unauthorised message.

In any case, I am forcing unauth rendering on the catch as this way the code will be  easier to understand.